iree/base/CMakeLists.txt

```diff
@@ -131,6 +131,30 @@ iree_cc_test(
     absl::strings
 )
 
+iree_select_compiler_opts(DYNAMIC_LIBRARY_LINKOPTS
```

Comment:
_ prefix if private; though you could move this to common copts if it meant that this file became handled by bazel-to-cmake

Reply:
Just made it private via the _ prefix for now.